app/validators/routes_and_redirects_validator.rb

```diff
@@ -21,7 +27,7 @@ def validate(record)
       redirects_must_not_have_routes(record, routes)
       redirects_must_include_base_path(record, redirects)
     else
-      routes_must_include_base_path(record, routes)
+      routes_must_include_base_path(record, routes) unless is_unpublishing
```

Comment:
Could we make this simpler some how? 

I'm not sure why/if it is working on an unpublishing as wouldn't they fail the `return unless record.base_path.present?` guard?

It feels like this class should either know about unpublishings and editions (which kinda sucks) or maybe it should be tailored to the class and they use the shared logic of a different validator. What do you think?